### DISCUSSION TITLE
We should avoid publishing Lime3DS onto the Google Play Store, as that could make us a target for Nintendo if we aren't already within their crosshairs.

### DISCUSSION THREAD

**Discussion by TheSystemGuy1337** [azahar-emu/azahar#473] [General]
We've already seen what happened with Citra in the past, and we should take steps to avoid getting DMCA'd off the face of the planet. 

**Nintendo has already proven that they do not care if users obtain ROMS legally; they will still DMCA you. Putting Lime3DS on the Play Store adds significant risk as since Nintendo went out of their way to carpet bomb Citra and chances are we're already in their crosshairs.**

In short, don't publish Lime3DS to the Play Store. I don't think it's worth the risk of having all of our hard work wiped out in a day over. I don't mind if you publish it to the Play Store, but at the same time I really don't recommend it at all.

**Comment by OpenSauce04** [comment]
We've already come to the conclusion that Nintendo has no interest in going after 3DS emulation. The only reason that Citra was taken down was due to them being owned by the same company as Yuzu, which was the real target of their takedown. If Nintendo were going to do something, they would have already done it by now, and the Play Store isn't going to change that.

**Reply by TheSystemGuy1337** [reply]
But we should still be careful; we might not be target number one for Nintendo right now but down the road we may end up becoming target number one.
